### DISCUSSION TITLE
A Cannonball Bug?

### DISCUSSION THREAD

**Discussion by Kimet1970** [djyt/cannonball#94] [General]
An interesting strange feature (bug?) in Cannonball is the Maranello's Horse (Ferrari's horse logo) inverts his position during the race!
(I'm using the beta 0.34 version with the pedal inversion fixed)
**Is it a Cannonball bug? Or does it come from the original Outrun?**

**Comment by djyt** [comment]
A bug in my code, never! ahem. ;)

Nope, this is a 'feature' with the original game too. If you use my [sprite viewer](https://github.com/djyt/system16_sprite_viewer) on OutRun, you'll see the Ferrari is only stored facing one direction in the ROMs. The hardware flips the sprite.

**Reply by yepyepyep4711** [reply]
Nice one! I had never noticed :)
